.github/workflows/ci.yml

```diff
@@ -7,6 +7,26 @@ on:
     branches: [ main ]
 
 jobs:
+  arm_test:
```

Comment:
Could it be named as arm64_test? As arm_test may indicate it is 32-bit.

Reply:
renamed to `arm64_test`.